README.md

```diff
@@ -45,20 +45,52 @@ You can run the service with a mocked smart card as follows:
 MOCK_SHORT_VALUE="<short_value_json>" MOCK_LONG_VALUE_FILE="<path_to_file>" make run
 ```
 
-If using the `tests/electionSample.json` file, you may use the following commands:
+If configuring with the `tests/electionSample.json` file (see Clerk example below), you may use all the following examples.
 
-### Voter
-This will interpolate the current unix time stamp in seconds as the value of `c`.
+Note: timestamps (eg. `c`, `bp`, `uz`, etc.) are UTC Timestamps which are in seconds, not miliseconds.
+
+### Clerk
 ```
-MOCK_SHORT_VALUE="{\"t\":\"voter\",\"bs\":\"12\",\"pr\":\"23\",\"c\":$(date +%s)}" make run
+MOCK_SHORT_VALUE="{\"t\":\"clerk\",\"h\":\"blah\"}" MOCK_LONG_VALUE_FILE="tests/electionSample.json" make run
 ```
 
 ### Poll Worker
 ```
 MOCK_SHORT_VALUE="{\"t\":\"pollworker\",\"h\":\"blah\"}" make run
 ```
 
-### Clerk
+### Voter
+`$(date +%s)` will interpolate the current unix time stamp in seconds as the value of `c`.
+
+#### Newly Encoded Voter Card Ready to Vote
+
+Note: To have an expired voter card, manually update the `c` value to be 60 minutes (in seconds) or more in the past.
 ```
-MOCK_SHORT_VALUE="{\"t\":\"clerk\",\"h\":\"blah\"}" MOCK_LONG_VALUE_FILE="tests/electionSample.json" make run
+MOCK_SHORT_VALUE="{\"t\":\"voter\",\"bs\":\"12\",\"pr\":\"23\",\"c\":$(date +%s)}" make run
+```
+
+#### Voter Card with Votes. Ready for VxPrint.
+```
+MOCK_SHORT_VALUE="{\"t\":\"voter\",\"bs\":\"12\",\"pr\":\"23\",\"c\":$(date +%s),\"v\":{\"102\":\"yes\",\"president\":[{\"id\":\"court-blumhardt\",\"name\":\"Daniel Court and Amy Blumhardt\",\"partyId\":\"2\"}],\"senator\":[{\"id\":\"hewetson\",\"name\":\"Heather Hewetson\",\"partyId\":\"3\"}],\"representative-district-6\":[{\"id\":\"tawney\",\"name\":\"Glen Tawney\",\"partyId\":\"3\"}],\"governor\":[{\"id\":\"abcock\",\"name\":\"Barbara Adcock\",\"partyId\":\"3\"}],\"lieutenant-governor\":[{\"id\":\"norberg\",\"name\":\"Chris Norberg\",\"partyId\":\"0\"}],\"question-a\":\"yes\"}}" make run
```

Comment:
let's not do this now, we're about to move the `v` to the `longValue`.

Reply:
Agreed. Can you add a commit to this PR to update example with value for `MOCK_LONG_VALUE`?  I need this for local testing of VxPrint.